.ci/runChecks.sh

```diff
@@ -5,4 +5,4 @@ npm run build:node
 npm run lint:eslint
 npm run lint:prettier
 npx -ws license-check
-npx better-npm-audit audit --exclude 1096302,1098615
+npx better-npm-audit audit --exclude 1096302,1098615,1098618
```

Comment:
I see a misuse of audit excludes here: next time please check

1. `npm audit fix`, then
2. `npx better-npm-audit audit` (without the excludes)

this audit error *is* fixable, so we should not just exclude it.

Reply:
you can also wait for #208 to be merged, I already did it there.